service/src/main/java/com/commercetools/pspadapter/payone/util/PayoneHash.java

```diff
@@ -0,0 +1,17 @@
+package com.commercetools.pspadapter.payone.util;
+
+import com.google.common.base.Charsets;
+import com.google.common.hash.Hashing;
+
+
+public final class PayoneHash {
+
+    /**
+     * Currently Payone doesn't support SHA2. We should change the hashing method
+     * after they introduce support for SHA2
+     * */
```

Comment:
pls follow javadoc convention: describe what function does, income argument, returning value.

As part of the function description mention the implementation details about MD5 and SHA2 usage.

As an example take a look at [MoneyUtil#getTaxRateIfPresentOrZero](https://github.com/commercetools/commercetools-payone-integration/blob/dc42546644b142bcc6e8130e774bb40ecc65a514/service/src/main/java/com/commercetools/pspadapter/payone/util/MoneyUtil.java#L31) method

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/178/commits/b6737ba2b29cd31d7f599ebd042ca35bb374e063